utils/queryApiGateway.tsx

```diff
@@ -20,6 +20,54 @@ export interface ApiEnvVars {
   pfxPath: string
 }
 
+/**
```

Comment:
ESLint made me move these after the merge conflict touched the file. They are moved unchanged.